tests/test_aiooncue.py

```diff
@@ -1,24 +1,49 @@
 #!/usr/bin/env python
+# pylint: disable=protected-access
 
 """Tests for `aiooncue` package."""
 
+from unittest.mock import patch
 import pytest
 
-
-from aiooncue import Oncue
-
-
-@pytest.fixture
-def response():
-    """Sample pytest fixture.
-
-    See more at: http://doc.pytest.org/en/latest/fixture.html
-    """
-    # import requests
-    # return requests.get('https://github.com/audreyr/cookiecutter-pypackage')
-
-
-def test_content(response):
-    """Sample pytest test function with the pytest fixture as an argument."""
-    # from bs4 import BeautifulSoup
-    # assert 'GitHub' in BeautifulSoup(response.content).title.string
+import aiohttp
+
+
+from aiooncue import (
+    LOGIN_INVALID_PASSWORD,
+    LOGIN_INVALID_USERNAME,
+    LoginFailedException,
+    Oncue,
+    ServiceFailedException,
+)
+
+@pytest.mark.asyncio
+async def test_login():
+    """Tests login"""
+
+    # Successful Login
+    session = aiohttp.ClientSession()
```

Comment:
This needs to be closed at the end of the test or make it a fixture so it self closes

Reply:
Good catch.  I've update to pass None in 

```python
session: aiohttp.ClientSession = None
``` 

The code under tests really shouldn't be accessing the network, this way if it tries to it'll throw an exception.